_overviews/getting-started/index.md

```diff
@@ -156,15 +156,15 @@ def hello(): Unit =
 
 You can define a method with `def` keyword and mark it as a "main" method with the `@main` annotation, designating it as
 the entry point in program execution. The method's type is `Unit`, which means it does not return a value. `Unit`
-can be thought of as an analogue to `void` keyword found in other languages. The `println` method will print the `"Hello World!"`
+can be thought of as an analogue to `void` keyword found in other languages. The `println` method will print the `"Hello, World!"`
 string to standard output.
 
-To run the program, execute `scala-cli run hello.scala` command. The file will be compiled and executed, with console output
+To run the program, execute `scala run hello.scala` command. The file will be compiled and executed, with console output
 similar to following:
 ```
-$ scala-cli run hello.scala             
-Compiling project (Scala 3.4.2, JVM (20))
-Compiled project (Scala 3.4.2, JVM (20))
+$ scala run hello.scala             
+Compiling project (Scala 3.5.0, JVM (20))
```

Comment:
there's also a variable `{{site.scala-3-version}}` which can automatically update when we have a new release.